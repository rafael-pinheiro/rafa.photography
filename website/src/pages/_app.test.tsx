import renderer from 'react-test-renderer';
import App from './_app';

it('renders correctly', () => {
  const Component = (props) => <div {...props} />;
  const pageProps = {
    prop1: 'test',
    prop2: 'also test'
  };

  const tree = renderer
    .create(<App Component={Component} pageProps={pageProps} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
