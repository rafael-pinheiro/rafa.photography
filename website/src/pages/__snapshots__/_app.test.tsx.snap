// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="undefined "
  >
    <h1>
      Rafa
    </h1>
    <nav>
      <a
        href="/"
      >
        <span>
          Home
        </span>
        <small>
          Back to homepage
        </small>
      </a>
      <a
        href="/"
      >
        <span>
          About
        </span>
        <small>
          Learn more about the author
        </small>
      </a>
      <a
        href="/"
      >
        <span>
          Contact
        </span>
        <small>
          Get in touch
        </small>
      </a>
    </nav>
    <button
      onClick={[Function]}
    >
      <span>
        menu
      </span>
      <span>
        <span />
        <span />
        <span />
      </span>
    </button>
  </div>,
  <div
    prop1="test"
    prop2="also test"
  />,
]
`;
